protected/humhub/modules/user/views/auth/login.php

```diff
@@ -56,12 +57,14 @@
                     <div class="col-md-4">
                         <?= Html::submitButton(Yii::t('UserModule.auth', 'Sign in'), ['id' => 'login-button', 'data-ui-loader' => "", 'class' => 'btn btn-large btn-primary']); ?>
                     </div>
+                    <?php if ($passwordRecoveryRoute) : ?>
                     <div class="col-md-8 text-right">
                         <small>
-                            <a id="password-recovery-link" href="<?= Url::toRoute('/user/password-recovery'); ?>"
+                            <a id="password-recovery-link" href="<?= Url::to($passwordRecoveryRoute); ?>"
```

Comment:
@gevorgmansuryan We should perhaps refactor this directly and use "Html::a". Ideally, external links (i.e. a string as a route) will also open in a new window.